habitat_baselines/evaluate_ppo.py

```diff
@@ -184,30 +180,34 @@ def eval_checkpoint(checkpoint_path, args, writer, cur_ckpt_idx=0):
             device=device,
         )
 
-        for i in range(not_done_masks.shape[0]):
-            if not_done_masks[i].item() == 0:
-                episode_spls[i] += infos[i]["spl"]
-                if infos[i]["spl"] > 0:
-                    episode_success[i] += 1
-
         rewards = torch.tensor(
             rewards, dtype=torch.float, device=device
         ).unsqueeze(1)
         current_episode_reward += rewards
-        episode_rewards += (1 - not_done_masks) * current_episode_reward
-        episode_counts += 1 - not_done_masks
-        current_episode_reward *= not_done_masks
-
         next_episodes = envs.current_episodes()
         envs_to_pause = []
         n_envs = envs.num_envs
         for i in range(n_envs):
-            if next_episodes[i].episode_id in stats_episodes:
+            if (
+                next_episodes[i].scene_id,
+                next_episodes[i].episode_id,
+            ) in stats_episodes:
```

Comment:
As I understand we will send for already paused environments `pause` command again. I think that's ok, but just double checking.

Reply:
I don't think it will. It will only track environments that's not paused, i.e. `envs.num_envs` decreases after pause.

Reply:
Yep, pause will only be sent at most once to each environment

Reply:
Got it, we are removing env from active envs once it's paused.